setup.py

```diff
@@ -0,0 +1,98 @@
+import os
+import re
+from pathlib import Path
+
+from setuptools import find_packages, setup
+
+NAME = "ptools_bin"
+PACKAGES = find_packages()
+META_PATH = Path("ptools_bin", "__init__.py")
+PROJECT_URLS = {
+    "Source Code": "https://github.com/ENCODE-DCC/ptools_bin",
+    "Issue Tracker": "https://github.com/ENCODE-DCC/ptools_bin/issues",
+}
+CLASSIFIERS = [
+    "License :: OSI Approved :: MIT License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: Implementation :: CPython",
+]
+INSTALL_REQUIRES = ["numpy==1.19.2", "biopython==1.78", "pandas==1.1.3"]
```

Comment:
How about making it `>=` instead of `==`? I would like to add it to Conda repo `bioconda` later. Fixing versions will result in many conflicts in Conda.

Reply:
Yeah, I will change that.